netjsonconfig/backends/zerotier/schema.py

```diff
@@ -0,0 +1,237 @@
+                                "ipRangeEnd": {
+                                    "type": "string",
+                                    "propertyOrder": 2,
+                                    "example": "192.168.192.254",
+                                    "description": "The ending IP address of the pool range",
+                                },
+                            },
+                        },
+                        "description": "Range of IP addresses for the auto assign pool",
+                    },
+                    "dns": {
+                        "type": "object",
+                        "propertyOrder": 16,
+                        "properties": {
+                            "domain": {
+                                "type": "string",
+                                "propertyOrder": 1,
+                                "example": "zerotier.openwisp.io",
+                                "description": "The domain for DNS resolution",
+                            },
+                            "servers": {
+                                "type": "array",
+                                "propertyOrder": 2,
+                                "items": {
+                                    "type": "string",
+                                    "example": "10.147.20.3",
+                                    "description": "The DNS server IP addresses",
+                                },
+                            },
+                        },
+                    },
+                    "rules": {
+                        "type": "array",
+                        "items": {"type": "object"},
+                        "propertyOrder": 17,
+                        "description": "Array of network rule objects",
+                    },
+                    "capabilities": {
+                        "type": "array",
+                        "items": {"type": "object"},
+                        "propertyOrder": 18,
+                        "description": "Array of network capabilities",
+                    },
+                    "tags": {
+                        "type": "array",
+                        "items": {"type": "object"},
+                        "propertyOrder": 19,
+                        "description": "Array of network tag objects",
+                    },
+                    "remoteTraceTarget": {
+                        "type": "string",
+                        "propertyOrder": 20,
+                        "example": "7f5d90eb87",
+                        "description": "The remote target ID for network tracing",
+                    },
+                    "remoteTraceLevel": {
+                        "type": "integer",
+                        "propertyOrder": 21,
+                        "description": "The level of network tracing",
+                    },
+                },
+            },
+        },
+    },
+}
+
+schema = deepcopy(base_zerotier_schema)
```

Comment:
why do we need to deepcopy the base schema and modify some properties below here? Is the base schema used anywhere else on its own without these modifications?